docs/src/main/asciidoc/_elytron/components/Distributed_Security_Realm.adoc

```diff
@@ -0,0 +1,34 @@
+[[distributed-realm]]
+= Distributed Realm
+
+The distributed realm allows combining identities located across multiple security realms of any type into a single security realm.
+
+The `distributed-realm` resource contains the following attributes: -
+
+* `realms` - List of realms in the order they should be queried
+* `ignore-unavailable-realm` - Whether subsequent realms should be checked after an unavailable realm is reached. The default value is false.
+* `emit-events` - Whether a SecurityEvent signifying realm unavailability should be emitted. The default value is true.
+
+== Defining a Distributed Realm
+
+Assuming realms `properties-realm`, `ldap-realm` and `jdbc-realm` already exist then `distributed-realm` combining these three can be created with the following command.
+
+[source,options="nowrap"]
+----
+/subsystem=elytron/distributed-realm=distributedRealmExample:add(realms=[properties-realm, ldap-realm, jdbc-realm])
+----
+
+When, for example, `ldap-realm` is unavailable, adding `ignore-unavailable-realm` will ensure that subsequent realms will be checked.
```

Comment:
Maybe this could be reworded to something like:

When configuring a `distributed-realm`, it is possible to ignore any unavailable realms by setting `ignore-unavailable-realms` to `true`. When retrieving an identity from a `distributed-realm`, each of the configured realms is invoked sequentially until a realm containing the identity is found. If a security realm becomes unavailable for some reason, if `ignore-unavailable-realms` to `true`, any subsequent realms will still be checked.